cea/technologies/network_layout/AGENTS.md

```
@@ -3,6 +3,160 @@
+### Input Format Support
+
+Both input formats fully supported:
+- **Shapefiles:** `edges-shp-path` + `nodes-shp-path`
+- **GeoJSON:** `network-geojson-path` (Point + LineString features)
+
+All modes work identically for both - `load_user_defined_network()` normalises to GeoDataFrames.
+
+### Example Console Outputs
+
+**Validate mode (missing buildings - ERROR):**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)